development-docs/DEV_GUIDE.md

```diff
@@ -382,20 +382,33 @@ the commit if errors are detected:
 ./tools/git-hooks/checkstyle-pre-commit
 ```
 
-## Building container images for other platforms with Docker `buildx`
+## Building container images for other platforms
 
+* build with default `build` command
+When building Strimzi images, it'll add some binaries into the images based on the target platform. The default 
+binaries will be loaded with `linux/amd64` platform compatible. If you want to change it, you can set the environment
+variable `DOCKER_BUILD_ARGS` to pass additional build options such as the platform and set the `TARGETPLATFORM` argument. 
+For example, following commands can be used to build Strimzi images for Linux Arm64 / AArch64 platform:
+
+```
+export DOCKER_BUILD_ARGS="--platform linux/arm64 --build-arg TARGETPLATFORM=linux/arm64"
```

Comment:
isn't `TARGETPLATFORM` set automatically when user specifies `--platform` for docker command?

Reply:
1. from my experience, it didn't set the `TARGETPLATFORM` automatically.
2. From the code, you can see no places set this argument. The docker arg should set explicitly using `--build-arg` IIRC.

Reply:
Ah, I didn't know that. I'll check it tomorrow.

Reply:
@scholzj @Frawless , you're right! The `TARGETPLATFORM` is set automatically via docker/podman. I found the issue why I failed, it's because in the podman in M1 macbook, the `TARGETPLATFORM` is set as `linux/arm64/v8`, instead of `linux/arm64` we expected. Updating the PR title and PR description and the changes. Please take a look again. Thank you.